loader.qml

```diff
@@ -97,12 +233,20 @@ ApplicationWindow {
 
         Item {
             id: wallClock
-            property var time: new Date()
+            property var time: timewidget(settime.checked)
             Timer {
                 interval: 1000
                 running: true
                 repeat: true
-                onTriggered: wallClock.time = new Date()
+                onTriggered: wallClock.time = wallClock.timewidget(settime.checked)
+            }
+            function timewidget(statictime) {
+                var mydate = new Date()
```

Comment:
Here too, use camel case and don't use the my prefix which does not mean anything.